src/test/v2.0/advanced/fdc3.raiseIntent-NoAppsFound.ts

```diff
@@ -0,0 +1,101 @@
+      }
+    });
+
+    const RaiseIntentFailTargetedAppInstanceResolve2 =
+      "(RaiseIntentFailTargetedAppInstanceResolve2) Should fail to raise intent when targeted app intent-a, context 'testContextY', intent 'aTestingIntent' and AppIdentifier IntentAppAId with instanceId property NonExistentInstanceId do not correlate";
+    it(RaiseIntentFailTargetedAppInstanceResolve2, async () => {
+      try {
+        await control.raiseIntent("aTestingIntent", "testContextY", {
+          appId: "IntentAppAId",
+          instanceId: "NonExistentInstanceId",
+        });
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        expect(ex).to.have.property("message", ResolveError.NoAppsFound);
+      }
+    });
+
+    const RaiseIntentFailTargetedAppResolve1 = "(RaiseIntentFailTargetedAppResolve1) Should fail to raise intent when targeted app intent-a, context 'testContextY', intent 'aTestingIntent' and AppIdentifier IntentAppAId do not correlate";
+    it(RaiseIntentFailTargetedAppResolve1, async () => {
+      try {
+        await control.raiseIntent("aTestingIntent", "testContextY", { appId: "IntentAppAId" });
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.NoAppsFound);
+      }
+    });
+
+    const RaiseIntentFailTargetedAppResolve2 = "(RaiseIntentFailTargetedAppResolve2) Should fail to raise intent when targeted app intent-a, context 'testContextY', intent 'aTestingIntent' and AppIdentifier NonExistentApp do not correlate";
+    it(RaiseIntentFailTargetedAppResolve2, async () => {
+      try {
+        await control.raiseIntent("aTestingIntent", "testContextY", { appId: "NonExistentApp" });
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.NoAppsFound);
+      }
+    });
+
+    const RaiseIntentFailTargetedAppResolve3 = "(RaiseIntentFailTargetedAppResolve3) Should fail to raise intent when targeted app intent-h, context 'testContextY', intent 'sharedTestingIntent2' and AppIdentifier IntentAppHId do not correlate";
+    it(RaiseIntentFailTargetedAppResolve3, async () => {
+      const { timeout, promise } = sleep(constants.NoListenerTimeout);
+
+      try {
+        await control.raiseIntent("sharedTestingIntent2", "testContextY", { appId: "NonExistentApp" });
+        await promise;
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        clearTimeout(timeout);
+        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.NoAppsFound);
+      }
+    }).timeout(constants.NoListenerTimeout + 1000);
+
+    const RaiseIntentFailTargetedAppResolve4 = "(RaiseIntentFailTargetedAppResolve4) Should fail to raise intent when targeted app intent-i, context 'testContextY', intent 'sharedTestingIntent2' and AppIdentifier IntentAppIId do not correlate";
```

Comment:
This is not a good description: context 'testContextY', intent 'sharedTestingIntent2' and AppIdentifier IntentAppIId  DO correlate in teh app directory, but the app adds the wrong listener type. I.e. it should resolve correctly, but then error out with the expected error (IntentDeliveryFailed, see below)

Reply:
Also I note that Intent-I IS adding the intent listener `sharedTestingIntent2` where this test needs it to add a different one and NOT that one.